x-pack/plugins/security_solution/server/lib/detection_engine/rules/get_prepackaged_rules.ts

```diff
@@ -16,10 +16,27 @@ import {
   AddPrepackagedRulesSchemaDecoded,
 } from '../../../../common/detection_engine/schemas/request/add_prepackaged_rules_schema';
 import { BadRequestError } from '../errors/bad_request_error';
+import {
+  getAsset,
+  getPathParts,
+  getRegistryPackage,
+  fetchFindLatestPackage,
+} from '../../../../../fleet/server';
 
-// TODO: convert rules files to TS and add explicit type definitions
 import { rawRules } from './prepackaged_rules';
 
+const DetectionRulesPackageName = 'detection_rules';
```

Comment:
usually constants are capital letters screaming such as `DETECTION_RULES_PACKAGE_NAME = 'detection_rules'` as a convention or they're just lowercase like `detectionRulesPackageName`.

We have a global constants file that is used front and backend here:

```ts
x-pack/plugins/security_solution/common/constants.ts
```

But some people prefer to put their constants in a file called `constants.ts` that is local and next to their file if they don't think it will be shared outside of their code. I'm fine with constants local to a file like you have it or any of the above. Putting it in the common section will add more bytes to the plugin downloaded to the front end so it might be better to keep it local to the file or just in another `constants.ts` next to this one.

Context fwiw.

Reply:
doh! yeah, I had a moment from Go, which yells at you if you have a global constant that doesn't follow the same naming convention as as a variable (it's dumb).

I added a `constants.ts` next to this file and moved this there
b301d4aceec65400844644207b436656ccb40040